opencv/src/OpenCV/ImgProc/GeometricImgTransform.hsc

```diff
@@ -458,6 +459,26 @@ getPerspectiveTransform srcPts dstPts = unsafeCoerceMat $ unsafePerformIO $
             );
         }|]
 
+{- | Calculates an affine transformation matrix for 2D affine transform
+
+<http://docs.opencv.org/3.0-last-rst/modules/imgproc/doc/geometric_transformations.html#getaffinetransform OpenCV Sphinx doc>
+-}
+getAffineTransform
+    :: (IsPoint2 point2 CFloat)
+    => V.Vector (point2 CFloat) -- ^ Array of 3 floating-point Points representing vertices in source image
```

Comment:
The `V.Vector` is a good Haskell synonym for the C++ `InputArray`. But the documentation specifies that it should be an array of exactly 3 points. In fact, [the C++ code](https://github.com/opencv/opencv/blob/master/modules/imgproc/src/imgwarp.cpp#L3164) contains the following assertions:
```c++
Mat src = _src.getMat(), dst = _dst.getMat();
CV_Assert(src.checkVector(2, CV_32F) == 3 && dst.checkVector(2, CV_32F) == 3);
```
So it will throw an exception if the input arrays are not of length 3.
Why not choose a type which guarantees that we always have 3 points? Something like `V3 (point2 CFloat)`. (_V3: A 3-dimensional vector_)